lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -228,8 +228,12 @@ html.video-fullscreen {
         }
       }
 
-      .proctored-exam-select-code {
-        @include margin-left(30px);
+      .copy-exam-code-button {
+        margin: 5px 0px;
```

Comment:
I couldn't get it to look perfect in all the browsers, so I favored Chrome.

Reply:
What behaved inconsistently?

Reply:
The button doesn't line up perfectly with the input in Firefox. Here's a picture. I have to squint to see it.

![image](https://user-images.githubusercontent.com/11871801/54155826-5311ec00-441b-11e9-9388-5f16c0f965fc.png)

`padding: 8px 20px;` aligns the two elements correctly on Firefox but not on Chrome and Safari.

P.S. I know our actual support of Safari is unclear at the moment, but I'm erring on the side of caution until that's resolved.

Reply:
That all sounds _great_. Esp. assuming support for Safari is still needed 💯